tests/ts/query-builder-api/eager-loading-methods.ts

```diff
@@ -0,0 +1,74 @@
+import { Person } from '../fixtures/person';
+
+(async () => {
+  await Person.query()
+    .where('firstName', 'Arnold')
+    .withGraphFetched('pets');
+
+  await Person.query().withGraphFetched('children.[pets, movies]');
+
+  await Person.query().withGraphFetched({
+    children: {
+      pets: true,
+      movies: true
+    }
+  });
+
+  await Person.query()
+    .withGraphFetched('children(selectNameAndId).[pets(onlyDogs, orderByName), movies]')
+    .modifiers({
+      selectNameAndId(builder) {
```

Comment:
I wonder if we can do better here with the `builder` typings. It is currently `any`.

Reply:
Really? Yep, we should definitely be able to do better. I can take a look.